Update seeds.rb

sdmmssa opened the pull request:
Added module definition

# PR Description

✨ This PR fixes the issue number: 6

✨ Here's the summary of my changes and my motivation behind them:

-Added definition for the word "module" and hopefully it is easy to understand
-
-

✨ This PR changes or adds functionality which will require updating the documentation: NO (leave what's relevant)


# Self-check:

Please check all that apply:

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my code, refactored it to the best of my abilities and deleted all unnecessary code
- [ ] I have commented my code, particularly in hard-to-understand areas; my comments are concise
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings and no new bugs


✨ ✨ ✨  Thank you for contributing to the Code Bending Dictionary ✨ ✨ ✨

sylwiavargas committed:
Dev (#29)

* updates word (#26)

* updates favorite (#27)

* updates readme to include contribution guide (#28)

* updates readme to include contribution guide

* updates readme to include contribution guide

* changes

Co-authored-by: Annie Zheng <24644341+bigfishh@users.noreply.github.com>

sdmmssa committed:
Update seeds.rb

Added module definition

sylwiavargas reviewed (changes requested):
Hi @sdmmssa ! Thank you for this contribution! I really love this analogy! ✨

I have a request for two quick changes: 
1. Since the word "module" for some reason did not make it to our list, would you be so kind to create it in the "Words" section (line 23 onwards) in the `seeds.rb` file?
2.  Since "module" exists both in Ruby and JS, could you add a second definition variable for Ruby(`module_def_ruby`), in which you'd change the word "function" to "method", and rename the current one to `module_def_js`? 
**context:** we are adding the tech attribute to the definition soon so I'm already thinking ahead :)

sdmmssa committed:
Update seeds.rb

Update module definition to be split into two with one being for js and the other for ruby. Changed "functions" to "methods" for ruby.
Added module to creating words

sdmmssa commented:
Hi,
I updated with the requested changes. Hopefully this is what you're asking for.

Thanks

sylwiavargas committed:
Merge branch 'dev' into main

sylwiavargas commented:
Hey @sdmmssa — I was about to add you to our contributors list in the READme, would you like that? 
If so:
1) Would you give me please your name or nickname (sdmmssa is also okay if that's your wish!) 
2) Is there a picture you want featured (github avatar is also ok!)

And, additionally — would you like the definition to be attributed to you?
